CineasteTests/SearchMoviesCellTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  SearchMoviesCellTests.swift
+//  CineasteTests
+//
+//  Created by Felizia Bernutz on 27.12.17.
+//  Copyright © 2017 notimeforthat.org. All rights reserved.
+//
+
+import XCTest
+@testable import Cineaste
+
+class SearchMoviesCellTests: XCTestCase {
+    let cell = SearchMoviesCell()
+    
+    override func setUp() {
+        super.setUp()
+
+        let imageView = UIImageView()
+        cell.addSubview(imageView)
+        cell.posterImageView = imageView
+
+        let title = UILabel()
+        cell.addSubview(title)
+        cell.movieTitleLabel = title
+    }
+    
+    func testConfigure() {
```

Comment:
I have chosen the name `testConfigure()` for the test because it tests a function called `configure` and I always try to name a test like the function and adding `test` in front of it.